### ISSUE TITLE
Add Content :  Build Model in PyTorch

### ISSUE THREAD

**Issue by kRiShNa-429407** [animator/learn-python#795]
In this I will replicate a standard linear regression model using pure PyTorch.

I'll use linear regression to create the data with known parameters (things that can be learned by a model) and then we'll use PyTorch to see if we can build model to estimate these parameters using gradient descent.

**Comment by kRiShNa-429407** [comment]
Please assign it to me

**Comment by kRiShNa-429407** [comment]
Sure
